### PR TITLE
Feat imporve cli

### PR DESCRIPTION
**Pull request by jsdidierlaurent**

### CODE REVIEW

File: cli/commands/version/version.go [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+	"github.com/monitoror/monitoror/pkg/templates"
+
+	"github.com/spf13/cobra"
+)
+
+const versionTemplate = ` Version:    {{green .Version}}{{ with .BuildTags }}{{printf " (%s)" . | grey }}{{end}}
+ Git commit: {{green .GitCommit}}
+ Built:      {{green .BuildTime}}
+
+ Go version: {{blue .GoVersion}}
```
Comment (Alex-D) [inline cli/commands/version/version.go:18]: Useless?

File: cli/helper/printer.go [hunk 2]
```diff
@@ -0,0 +1,226 @@
...
+
+	"github.com/monitoror/monitoror/cli"
+	"github.com/monitoror/monitoror/cli/version"
+	coreConfig "github.com/monitoror/monitoror/config"
+	coreModels "github.com/monitoror/monitoror/models"
+	"github.com/monitoror/monitoror/pkg/system"
+	"github.com/monitoror/monitoror/pkg/templates"
+)
+
+var monitororTemplate = `
```
Comment (Alex-D) [inline cli/helper/printer.go:17]: startupTemplate maybe?

File: cmd/monitoror/monitoror.go [hunk 3]
```diff
@@ -1,37 +1,81 @@
...
+				return err
+			}
+			return server.Start()
+		},
+		Version:       fmt.Sprintf("%s, build %s", version.Version, version.GitCommit),
+		SilenceUsage:  true,
+		SilenceErrors: true,
+	}
+
+	cmd.Flags().BoolP("version", "v", false, "Print version information and quit")
```
Comment (Alex-D) [inline cmd/monitoror/monitoror.go:43]: "and quit" seems unnecessary

Comment (Alex-D) [reply cmd/monitoror/monitoror.go:43]: "Print the version number"

File: .env.example [hunk 4]
```diff
@@ -1,49 +1,49 @@
...
-MO_DOWNSTREAMCACHEEXPIRATION=120000
-MO_INITIALMAXDELAY=1700
+#MO_PORT=8080
+#MO_UPSTREAMCACHEEXPIRATION=10000
+#MO_DOWNSTREAMCACHEEXPIRATION=120000
+#MO_INITIALMAXDELAY=1700
 
 # Config
-MO_CONFIG=./config-example.json
+#MO_CONFIG=./config-example.json
```
Comment (Alex-D) [inline .env.example:7]: Maybe keep this one un-commented to make it work when init?

Comment (Alex-D) [reply .env.example:7]: Or better: remove the comment on init

Comment (Alex-D) [reply .env.example:7]: Should be `config.json` according to the init function

Comment (Alex-D) [reply .env.example:7]: So: keep it like that in example, but at the time you create the config.json, make sure that this line is updated to:

    MO_CONFIG=./config.json 

in the newly created `.env`

File: cli/commands/version/version.go (hunk 1 above)
Comment (jsdidierlaurent) [reply cli/commands/version/version.go:18]: `monitoror version`  command print all build information
`monitoror --version` build only monitoror version

It's two different needs (like docker, kubernetes, and many other does)

File: cmd/monitoror/monitoror.go (hunk 3 above)
Comment (jsdidierlaurent) [reply cmd/monitoror/monitoror.go:43]: "and quit" explains this argument exit monitoror, not like the other arguments

File: .env.example (hunk 4 above)
Comment (jsdidierlaurent) [reply .env.example:7]: Done

File: cmd/monitoror/monitoror.go (hunk 3 above)
Comment (Alex-D) [reply cmd/monitoror/monitoror.go:43]: Like all -v or --version commands? :D

**Pull request merged**